libs/wire-api-federation/src/Wire/API/Federation/API/Brig.hs

```diff
@@ -44,7 +44,7 @@ import Wire.API.UserMap (UserMap)
 import Wire.API.Util.Aeson (CustomEncoded (..))
 import Wire.Arbitrary (GenericUniform (..))
 
-newtype SearchRequest = SearchRequest {term :: Text}
+data SearchRequest = SearchRequest {term :: Text, from :: Maybe TeamId, onlyInTeams :: Maybe [TeamId]}
```

Comment:
Can you please break this into multiple lines and add Haddock for the new fields?